src/utils/etherspot.js

```diff
@@ -70,6 +70,13 @@ const ETHERSPOT_TRANSACTION_HISTORY_STATUS = {
   REVERTED: 'Reverted',
 };
 
+export const parseEtherspotTransactionStatus = (transactionStatus: string) => {
+  if (transactionStatus === ETHERSPOT_TRANSACTION_HISTORY_STATUS.COMPLETED) return TRANSACTION_STATUS.CONFIRMED;
+  if (transactionStatus === ETHERSPOT_TRANSACTION_HISTORY_STATUS.REVERTED) return TRANSACTION_STATUS.FAILED;
+
+  return TRANSACTION_STATUS.PENDING;
```

Comment:
Do we somehow support timeout status? If etherspot does not return one, could we remove `TRANSACTION_STATUS.TIMEDOUT` on our end?

Reply:
On Etherspot itself it doesn't exist, but there are existing mappings for Archanova so would keep as long as we support it.

Reply:
Makes sense